src/parser/jobs/gnb/modules/SonicBreak.tsx

```diff
@@ -44,9 +46,8 @@ export class SonicBreak extends Analyser {
 
 	override initialise() {
 
-		if (this.parser.patch.after('7.3')) {
-			MAX_TICKS = 5
-		}
+		MAX_TICKS = this.parser.patch.before('7.4') ? MAX_TICKS_PRE_7_4 : MAX_TICKS_POST_7_4
```

Comment:
One final nit: Can we move MAX_TICKS inside of the Sonic Break class as a private maxTicks variable, to separate the consts from the assignable variables?